### PR TITLE
Show notification when new version is available

### PR DESCRIPTION
**Pull request by therealssj**
Preview of the change:
screen shot 2018-01-05 at 2 49 27 am

Resolves:#815

### CODE REVIEW

File: src/gui/static/src/app/app.component.ts [hunk 1]
```diff
@@ -53,7 +62,15 @@ export class AppComponent {
   }
 
   private setVersion() {
-    return this.apiService.get('version')
+    // Set build version
+    this.apiService.get('version')
       .subscribe(output => this.version = output.version);
+
+    // Set latest release version from github
+    this.http.get('https://api.github.com/repos/skycoin/skycoin/tags')
```
Comment (gz-c) [inline src/gui/static/src/app/app.component.ts]: this should ignore tags that match *-rc*

File: src/gui/static/src/app/app.component.ts [hunk 2]
```diff
@@ -1,32 +1,10 @@
-import { Component } from '@angular/core';
-import { WalletService } from './services/wallet.service';
-import { BlockchainService } from './services/blockchain.service';
+import {Component} from '@angular/core';
```
Comment (samuelvisscher) [inline src/gui/static/src/app/app.component.ts]: Maintain the code style please

File: src/gui/static/src/app/components/layout/header/header.component.ts [hunk 3]
```diff
@@ -1,8 +1,11 @@
-import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import {Component, Input, OnDestroy, OnInit} from '@angular/core';
```
Comment (samuelvisscher) [inline src/gui/static/src/app/components/layout/header/header.component.ts]: Same here, maintain code style please

File: src/gui/static/src/app/components/layout/header/header.component.ts [hunk 4]
```diff
@@ -58,4 +67,29 @@ export class HeaderComponent implements OnInit, OnDestroy {
...
+      .subscribe(output =>  this.version = output.version);
+
+    // Set latest release version from github
+    this.http.get('https://api.github.com/repos/skycoin/skycoin/tags')
+      .map((res: any) => res.json())
+      .catch((error: any) => Observable.throw(error || 'Unable to fetch latest release version from github.'))
+      .subscribe(response =>  {
+        // Iterate though the tags
+        // Find the latest tag which is not a rc
+        for ( const i in response ) {
```
Comment (samuelvisscher) [inline src/gui/static/src/app/components/layout/header/header.component.ts]: Use ES6 .find function here

**Comment by gz-c** [comment]
@samuelvisscher check again

**Comment by gz-c** [comment]
Need to test this when the latest tag is an `-rc` tag

**Comment by therealssj** [comment]
@gz-c https://github.com/skycoin/skycoin/pull/821/files#diff-5d6523bfde2dd31deac52628d2d13c94R82

the this line returns the first element which is not `-rc` tag

**Pull request merged**